pkg/queue/cluster_queue_impl.go

```diff
@@ -35,14 +35,17 @@ import (
 // ClusterQueueImpl is the base implementation of ClusterQueue interface.
 // It can be inherited and overwritten by other class.
 type ClusterQueueImpl struct {
-	// QueueingStrategy indicates the queueing strategy of the workloads
-	// across the queues in this ClusterQueue.
-	QueueingStrategy kueue.QueueingStrategy
-
 	heap              heap.Heap
 	cohort            string
 	namespaceSelector labels.Selector
 
+	// popId identifies the last call to Pop. It's incremented when calling Pop.
```

Comment:
can we add something here to explain the purpose of those two parameters (something like tracking events that trigger re-queueing while a workload is being scheduled)

Reply:
Done